### ISSUE TITLE
Support for OID4VCI draft13?

### ISSUE THREAD

**Issue by kitakaze-kan** [OWND-Project/OWND-Wallet-iOS#26]
Are there any plans to support OID4VCI draft13?

**Comment by moratori** [comment]
@kitakaze-kan Yes, work is ongoing to make it compatible with [ID1 (draft 13)](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-ID1.html).
For iOS, I believe that a large part of it has been completed in pull request #17. 
Conformance with the latest specifications will be improved based on future test results, etc.

**Comment by kitakaze-kan** [comment]
Great, thanks for the confirmation!
